desktop/cmp/contextmenu/StoreContextMenu.js

```diff
@@ -137,6 +140,12 @@ export class StoreContextMenu {
                     hidden: !gridModel?.autosizeEnabled,
                     actionFn: () => gridModel.autosizeAsync()
                 });
+            case 'restoreDefaults':
+                return new RecordAction({
+                    text: 'Restore Grid Defaults',
+                    icon: Icon.reset(),
+                    actionFn: () => gridModel.restoreDefaults()
```

Comment:
Should the action have a built in confirm?

Reply:
I think so -- its a pretty rare and drastic action, and can also clarify that it is going to be *immediately* happening, which might help in the case that the button is clicked when commitOnChange is true